sim/dex.ts

```diff
@@ -141,8 +139,8 @@ export class ModdedDex {
 	readonly isBase: boolean;
 	readonly currentMod: string;
 
-	readonly getId: (text: any) => ID;
-	readonly getString: (str: any) => string;
+	readonly getString: (text: any) => string;
```

Comment:
Is this used, btw? Would it be easy to refactor this to `Utils`?

Reply:
It wouldn't be _too_ hard - Dex.getString is used in 10 spots. I can do that.